test/madetomeasure_api/test/routes/subscribers.clj

```diff
@@ -11,6 +11,14 @@
     (body body-string)
     (content-type "application/json")))
 
+(defn json-delete [endpoint]
+  (->
+    (request :delete endpoint)
+    (content-type "application/json")))
+
+(def subscriber
```

Comment:
https://clojure.github.io/clojure/clojure.test-api.html#clojure.test/compose-fixtures
https://clojure.github.io/clojure/clojure.test-api.html#clojure.test/join-fixtures
https://clojure.github.io/clojure/clojure.test-api.html#clojure.test/use-fixtures

I'm not sure how `use-fixtures` interacts with setup and teardown though.